modules/delegates.js

```diff
@@ -247,23 +238,26 @@ __private.forge = function(cb) {
 	const lastBlock = modules.blocks.lastBlock.get();
 
 	if (currentSlot === slots.getSlotNumber(lastBlock.timestamp)) {
-		library.logger.debug('Waiting for next delegate slot');
+		library.logger.debug('Block already forged for the current slot');
```

Comment:
I get your point but in this case the message `Waiting for next delegate slot` is not accurate.

I'm in favour of changing it and document it in the release notes.

@MaciejBaj What's your call?

Reply:
@lsilvs, the `Waiting for next delegate slot` is not really accurate in this step of the `forge` function. It also covers the same log message in the next step:
```
library.logger.debug('Waiting for delegate slot', {
			) {						currentSlot: slots.getSlotNumber(),
				library.logger.debug('Delegate slot', slots.getSlotNumber());					});
```
It's a `debug` level message, so it should not be qualified as breaking. 

`Block already forged for the current slot` might suggest a delegate already forged this particular block, whereas the check only covers if a correct block was applied for a given slot.

More accurate would be: `Block already applied for the current slot, skipping forging attempt.`

Reply:
@MaciejBaj I like `Block already forged for the current slot` log message more, the one you proposed is more confusing because we are not skipping forging attempt here as it's not our slot.

Reply:
Ok please leave the "Block already forged..."